src/supernode/DAPI_RPC_Server.cpp

```diff
@@ -71,7 +62,7 @@ bool supernode::DAPI_RPC_Server::HandleRequest(const epee::net_utils::http::http
     epee::serialization::storage_entry id_ = epee::serialization::storage_entry(std::string());
 
     if( !ps.load_from_json(query_info.m_body) ) {
-		LOG_PRINT_L5("!load_from_json");
+        LOG_PRINT_L0("!load_from_json");
```

Comment:
Maybe it's better to log errors to LOG_ERROR?

Reply:
Fixed